src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer start work");
+    }
+
+    public void stopWork() {
+        System.out.println("Bulldozer stop work");

```

Comment:
Similarly, the message 'Bulldozer stop work' could be more informative. Consider specifying what has been completed or the state of the Bulldozer. For instance, 'Bulldozer has finished clearing the construction site and is now off.'